app/styles/mixin/image-placeholder.scss

```diff
@@ -1,10 +1,8 @@
-// placeholder for setting up an image placeholder background
 %image-placeholder {
-	background: {
-		color: $gainsboro;
-		image: $brackets-svg-image;
-		position: center;
-		repeat: no-repeat;
-		size: contain;
-	}
+	color: $color-white;
```

Comment:
So, sometimes we're using $color-white and sometimes #fff. Let's use one way everywhere :)

Reply:
shouldn't we use `$wds-color-white` everywhere?